apps/industrial-hemp/views/authorised-witness-dbs-updates.html

```diff
@@ -0,0 +1,7 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#renderField}}authorised-witness-dbs-subscription{{/renderField}}
+    {{> partials-warning-dbs-subscription-responsible-officer}}
```

Comment:
I would suggest to rename this partials to make it more generic 

```suggestion
    {{> partials-warning-dbs-subscription}}
```

this would mean updating `site-responsible-officer-dbs-updates.html` template

Reply:
Yes absolutely, i've renamed this partials file so it can be generic.